guidance/be-nice.md

```diff
@@ -0,0 +1,53 @@
+# Contributors Code of Conduct
+
+The .NET Foundation was created to foster an open, innovative and inclusive
+community around open source .NET. To clarify expected behaviour in our
+communities we have adopted the [Contributor Covenant](http://www.dotnetfoundation.org/code-of-conduct).
+This code of conduct has been adopted by [many other open source communities](http://contributor-covenant.org/adopters/)
+and we feel it expresses our values well. 
+
+For a full copy of the [code of conduct](http://www.dotnetfoundation.org/code-of-conduct),
+see the .NET Foundation Website http://www.dotnetfoundation.org/code-of-conduct
+
+## Applying the Code of Conduct to your project
+
+First of all, it is important that you read the [code of conduct](http://www.dotnetfoundation.org/code-of-conduct)
+and understand when to be on the look out for unnaceptable behaviour, what to do if you see
+it or if someone reports their concerns to you.
+
+Any concerns passed to you by your community should be forwarded to the
```

Comment:
Re-reading the Contributor Covenant I agree. I'll tone it down but I'll also add a note in about people being able to email conduct if they are not happy with how a maintainer handled a situation.